File: codegens/dart/codesnippet.dart
import 'dart:convert' as convert;
import 'package:http/http.dart' as http;

void main() async {
const url = "https://mockbin.org/redirect/302/1/?to=https://postman-echo.com/get";
const payload = {};
Map<String, String> headers= {};

var response = await http.get(url,
	headers: headers,
);

print(response.body);
}

File: codegens/nodejs-axios/codesnippet.js
var axios = require('axios'),
  qs = require('qs');

var config = {
    'headers': {
    }
}

axios.get('https://mockbin.org/redirect/302/1/?to=https://postman-echo.com/get', config).then((response) => {
	console.log(JSON.stringify(response.data));
}).catch((err) => {
	console.log(err);
});
